anchore-ci/common_tasks

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env bash
+
+############################################################
+# Common CI tasks. These are intended to be reusable across
+# projects, and easily overriden.
+############################################################
+
+clean() {
+  TEST_IMAGE_NAME="${1:?'Missing required parameter: TEST_IMAGE_NAME.'}"
```

Comment:
All function local variables should use `local` when they're declared.

Reply:
Yepper!